plugins/like/lib/api/filters/KalturaLikeFilter.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/**
+ * @package plugins.like
+ * @subpackage api.filters
+ */
+class KalturaLikeFilter extends KalturaLikeBaseFilter
+{
+	/* (non-PHPdoc)
+	 * @see KalturaFilter::getCoreFilter()
+	 */
+	public function getCoreFilter()
+	{
+		return new LikeFilter();
+	} 
+	
+	/* (non-PHPdoc)
+	 * @see KalturaRelatedFilter::getListResponse()
+	 */
+	public function getListResponse(KalturaFilterPager $pager, KalturaDetachedResponseProfile $responseProfile = null)
```

Comment:
This class doesn't extend KalturaRelatedFilter, so either extend it or remove this method implementation.